internal/server/create_document_digital_lpa.go

```diff
@@ -45,9 +47,17 @@ func CreateDocumentDigitalLpa(client CreateDocumentDigitalLpaClient, tmpl templa
 			XSRFToken: ctx.XSRFToken,
 		}
 
-		group, groupCtx := errgroup.WithContext(ctx.Context)
+		data.Lpa, err = client.DigitalLpa(ctx, uid)
```

Comment:
Might be worth creating a shared function that returns `CaseTabData` so you don't have to fetch/compile the bits in each endpoint handler (or mock them each in tests).

Reply:
Yeah, something like that. Probably easier as a follow-up ticket to give some space to explore what's possible.

Reply:
I think it will be relatively easy to parcel up the digital LPA and task list into one structure, and provide a client method to fetch that. I think I'll do that on this ticket, as I was unhappy about the amount of repetition and busy work the current approach took.

Reply:
We could return to some more generic way to group HTTP calls in another ticket maybe. It feels like we have a lot of repetition there, with multiple HTTP calls being made in groups, sometimes multiple times in the same handler (edit document is particularly bad for this).